Source/qol/stash.cpp

```diff
@@ -95,11 +95,10 @@ void CheckStashPaste(Point cursorPosition)
 
 	const Size itemSize = GetInventorySize(player.HoldItem);
 	const Displacement hotPixelOffset = Displacement(itemSize * INV_SLOT_HALF_SIZE_PX);
-	if (IsHardwareCursor()) {
-		// It's more natural to select the top left cell of the region the sprite is overlapping when putting an item
-		//  into an inventory grid, so compensate for the adjusted hot pixel of hardware cursors.
-		cursorPosition -= hotPixelOffset;
-	}
+
+	// It's more natural to select the top left cell of the region the sprite is overlapping when putting an item
+	//  into an inventory grid, so compensate for the adjusted hot pixel of hardware cursors.

```

Comment:
```suggestion
	//  into an inventory grid, so compensate for the adjusted hot pixel of item cursors.
```

I think this is the only reference to `hotPixelOffset` now as well, could inline the temporary?